### ISSUE TITLE
Node-notifier silently fails when using electron-forge

### ISSUE THREAD

**Issue by mangosing** [mikaelbr/node-notifier#358]
I created a vanilla electron forge project using their webpack script `npx create-electron-app my-new-app --template=webpack`. From there I followed the `node-notifier` docs and inserted the `notifier.notify('Message');` inside the `createWindow` function. I an console.log `noitifer` but and not getting any notification. Does anyone have an idea of what may be causing this? Is this being blocked by `electron-forge` somehow? Any and all help is much appreciated.

**Comment by hovancik** [comment]
duplicate of #352 ?

**Comment by mangosing** [comment]
@hovancik Im not sure this is exactly a duplicate. If you follow the `Quickstart` guide from the electron docs and then add in `node-notifier` in the main process it works. When you try to do the same thing even with a 'fresh-out-the-box' electron-forge install it silently fails from the main process but not the renderer process.
